app/components/framework/FrameworkImageButton/index.tsx

```diff
@@ -10,22 +11,36 @@ import {
     Kraken,
 } from '@the-deep/deep-ui';
 
-import { useRequest } from '#base/utils/restRequest';
+import {
+    AnalysisFrameworkQuery,
+    AnalysisFrameworkQueryVariables,
+} from '#generated/types';
+
 import { useModalState } from '#hooks/stateManagement';
-import { AnalyticalFramework } from '#types';
 import _ts from '#ts';
 
 import styles from './styles.css';
 
-const query = {
-    fields: ['title', 'preview_image', 'id'],
-};
+const ANALYSIS_FRAMEWORK = gql`
```

Comment:
Let's give it a better (more specific) name to avoid confusion with other Analysis Framework queries.